Packs/Tenable_io/TestPlaybooks/playbook-Tenable.io_test.yml

```diff
@@ -333,44 +333,6 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "9":
```

Comment:
why are these TPB steps deleted ?

Reply:
the previous step returns empty result, but it passes, this task checks as if there are results returned